Engine/source/console/simObject.cpp

```diff
@@ -2662,17 +2733,108 @@ DefineConsoleMethod( SimObject, getDynamicFieldCount, S32, (),,
 
 //-----------------------------------------------------------------------------
 
+DefineConsoleMethod( SimObject, getStaticField, const char*, ( S32 index ),,
+   "Get a value of a static field by index.\n"
+   "@param index The index of the static field.\n"
+   "@return The value of the static field at the given index or \"\"." )
+{
+	char fieldName[128];
+	char fieldValue[128];
+
+	const char* fn = object->getStaticField(index);
+
+	if(fn)
+		dStrcpy(fieldName, fn);
+	else
+		dStrcpy(fieldName, "");
```

Comment:
if the length of the field name or value is > 128, this will cause a buffer overflow. Similar case below.